WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/FeatureConfig/MLSMigrationFeatureConfigDecoder.swift

```diff
@@ -24,7 +24,7 @@ struct MLSMigrationFeatureConfigDecoder {
         from container: KeyedDecodingContainer<FeatureConfigEventCodingKeys>
     ) throws -> MLSMigrationFeatureConfig {
         let payload = try container.decode(
-            FeatureWithConfig<Payload>.self,
+            FeatureWithConfig<FeatureConfigResponse.MLSMigrationV6>.self,
```

Comment:
I'm guessing this was because the payloads were the same? Still, I'm not sure about mixing versioned and unversioned payloads, even if they're the same. What do you think?

Reply:
My bad, I forgot to undo my changes on this one.

Reply:
done thx